src/Indigo/Contracts/Common.hs

```diff
@@ -0,0 +1,16 @@
+-- SPDX-FileCopyrightText: 2020 TBD
+--
+-- SPDX-License-Identifier: LicenseRef-Proprietary
+{-# OPTIONS_GHC -Wno-orphans #-}
```

Comment:
IMO this should be used in modules as restricted as possible, this module is good in what it contains, but I'd move it to be `Indigo.Contracts.Common.Errors`.